src/rtcModule/OBJCCaptureModule.mm

```diff
@@ -0,0 +1,157 @@
+
```

Comment:
We may want to add this file to the .pro, so it appears in the project inside QtCreator (despite it will not be compiled). It may help to browse the code and to read it, even when its compilation requires xCode